brainglobe_workflows/cellfinder/cellfinder_main.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+
+import pooch
+from brainglobe_utils.IO.cells import save_cells
+from cellfinder_core.main import main as cellfinder_run
+from cellfinder_core.tools.IO import read_with_dask
+
+# Input data URL and hash
+DATA_URL = "https://gin.g-node.org/BrainGlobe/test-data/raw/master/cellfinder/cellfinder-test-data.zip"
+DATA_HASH = "b0ef53b1530e4fa3128fcc0a752d0751909eab129d701f384fc0ea5f138c5914"
+
+# Local cached directories
+CELLFINDER_CACHE_DIR = Path.home() / ".cellfinder_benchmarks"
+INPUT_DATA_CACHE_DIR = CELLFINDER_CACHE_DIR / "cellfinder_test_data"
```

Comment:
I think we need a way to be able to configure the value for `INPUT_DATA_CACHE_DIR`, so we can run the same workflow on different data?

I am thinking the best way is to check whether an environmental variable exists, and if it doesn't, use this data as default, via `pooch`?

Reply:
For an idea, see https://github.com/brainglobe/brainglobe-workflows/blob/8be0ed0cff5e0b06ef769faa05b44f1958d34679/brainglobe_workflows/registration_script.py#L76